webrtc-stats.html

```diff
@@ -1836,7 +1838,10 @@
               </dt>
               <dd>
                 <p>
-                  Represents the total number of connectivity check retransmissions received.
+                  Represents the total number of connectivity check request retransmissions
```

Comment:
Hm. This is a retransmissions *received* count, which means that the number of retransmitted requests received will reflect the number of retransmissions that the sender thinks is lost, and is driven by the remote end's desire for information.
The number of retransmitted responses received (strictly: the number of responses received that are in response to a retransmitted requests - responder side doesn't ever initiate retransmissions) is received at the side that initiates the requests, which means that it is always strictly less than the number of retransmitted requests sent (unless duplication happens), a number the receiver doesn't have access to.
I'm not sure either is very useful for diagnostics, but I'm very sure they're different.